src/main/kotlin/no/nav/klage/texts/service/TextService.kt

```diff
@@ -109,20 +114,9 @@ class TextService(
             ) ?: throw ClientErrorException("det må finnes en publisert versjon før et nytt utkast kan lages")
         }
 
-        val existingDraft = textVersionRepository.findByPublishedDateTimeIsNullAndTextId(
-            textId = textId
+        return textVersionRepository.save(
+            existingVersion.createDraft()
         )
-
-        return if (existingDraft != null) {
```

Comment:
Trengs ikke denne sjekken lenger?

Reply:
Det ble avtalt med FE at dette var for strengt. D.v.s. å skrive over eksisterende draft med nytt bare fordi du trykket på "create draft" en gang til. Det skal sies at FE nå også skjuler det alternativet når det allerede finnes et draft.

Reply:
Høres bra ut :+1: